stix_shifter/src/modules/splunk/README.md

```diff
@@ -0,0 +1,7 @@
+# Splunk
+
+### Format for calling stix-shifter from the command line
+
+python stix_shifter.py `<translator_module>` `<query or result>` `<stix identity object>` `<data>`
+
+(Note the identity object is only used when converting from AQL to STIX, but due to positional arguments, an empty hash will need to be passed in when converting from STIX patterns to AQL. Keyword arguments should be implemented to overcome this).
```

Comment:
This should probably be changed to read SPLUNK to STIX. I realize this was coming from a copy and paste. Ideally we could rework things so we don't need to pass in that empty hash, and so not even need the note.

Reply:
I've updated the PR and added content to the Readme in the Splunk Module.